AttackItem.h

```diff
@@ -1 +1,11 @@
-#include "Item.h"
\ No newline at end of file
+#pragma once
+
+#include "Item.h"
+#include "Character.h"
+#include "Utility.h"
```

Comment:
Don't include what you don't use
no member functions or member variables in `Character` are being used in this header file
- [ ] revise